tests/lib/devicetree/api/src/main.c

```diff
@@ -737,6 +744,13 @@ static void test_phandles(void)
 	zassert_true(DT_INST_PROP_HAS_IDX(0, gpios, 1), "");
 	zassert_false(DT_INST_PROP_HAS_IDX(0, gpios, 2), "");
 
+	/* DT_INST_PROP_HAS_NAME */
+	zassert_false(DT_INST_PROP_HAS_NAME(TEST_PH, foos, A), "");
+	zassert_true(DT_INST_PROP_HAS_NAME(TEST_PH, foos, a), "");
+	zassert_false(DT_INST_PROP_HAS_NAME(TEST_PH, foos, b-c), "");
+	zassert_true(DT_INST_PROP_HAS_NAME(TEST_PH, foos, b_c), "");
+	zassert_false(DT_INST_PROP_HAS_NAME(TEST_PH, bazs, jane), "");
```

Comment:
Looks like the fix for this got squashed into the wrong commit. It's fixed later on in ` tests/lib/devicetree: api: Add tests for new DT_{INST_}CLOCKS macros `, but it shouldn't be.

Reply:
Oh, thanks. Fixed